tests/Unit/Console/Commands/OpenApiCommandTest.php

```
@@ -78,4 +79,81 @@ public function testExecShowsErrorWhenModuleIsMissing(): void
+        @unlink($sourceDir . DS . 'index.html');
+        @rmdir($sourceDir);
+    }
+
+    public function testGenerateOpenapiSpecificationCreatesSpecFile(): void
+    {
+        $command = new class () extends OpenApiCommand {
+            public function info(string $message): void
+            {
+            }
+
+            public function error(string $message): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$message'. (undefined)

(UnusedFormalParameter)